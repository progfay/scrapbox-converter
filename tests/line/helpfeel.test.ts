/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('helpfeel', () => {
  it('Simple helpfeel', () => {
    const input = `Helpfeel
? Helpfeel notation`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
