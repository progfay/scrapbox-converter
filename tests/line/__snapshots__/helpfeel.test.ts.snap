// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helpfeel Simple helpfeel 1`] = `
"<article>
  <h1 class=\\"title\\">Helpfeel</h1>
  <section>
    <p><pre class=\\"helpfeel\\">Helpfeel&nbsp;notation</pre>
    </p>
  </section>
</article>"
`;
